recipes/xorg-xproto/bld.bat

```diff
@@ -0,0 +1,7 @@
+:: Just delegate to the Unixy script; but some of the X.org tarballs have
+:: config.guess files too ancient to recognize 64-bit Windows!
+curl -sSL -o config.guess "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess"
+curl -sSL -o config.sub "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub"
```

Comment:
Maybe we should download those and add to the recipe to ensure they won't change or disappear ?